code/_globalvars/lists/quirks.dm

```diff
@@ -98,3 +98,18 @@ GLOBAL_LIST_INIT(paraplegic_choice, list(
 	"Default" = FALSE,
 	"Amputee" = TRUE,
 ))
+
+///chipped Quirk
+GLOBAL_LIST_INIT(quirk_chipped_choice, list(
+	"Basketsoft 3000" = /obj/item/skillchip/basketweaving,
+	"WINE" = /obj/item/skillchip/wine_taster,
+	"Hedge 3" = /obj/item/skillchip/bonsai,
+	"Skillchip adapter" = /obj/item/skillchip/useless_adapter,
+	"N16H7M4R3" = /obj/item/skillchip/light_remover,
+	"3NTR41LS" = /obj/item/skillchip/entrails_reader,
+	"GENUINE ID Appraisal Now!" = /obj/item/skillchip/appraiser,
+	"Le S48R4G3" = /obj/item/skillchip/sabrage,
+	"Integrated Intuitive Thinking and Judging" = /obj/item/skillchip/intj,
+	"F0RC3 4DD1CT10N" = /obj/item/skillchip/drunken_brawler,
+	//"\"Space Station 13: The Musical\"" = /obj/item/skillchip/musical,
```

Comment:
I think it'd be nice to have this enabled, but that's just my personal opinion.

Also for a moment I was like "woo, fishing quirk" when you said that skillchips from (unhacked) toy vending machines could be chosen, but I get why you didn't include the fishing skillchip. It's generally quite more useful than anything on this list.

Reply:
Actually this pr was made before that. i'll go and enable'em